docs/support/managing-ckeditor-logo.md

```diff
@@ -0,0 +1,104 @@
+---
+category: licensing
+order: 30
+meta-description: Managing the "Powered by CKEditor" logo
+---
+
+# Managing the "Powered by CKEditor" logo
+
+## Why the "Powered by CKEditor" logo?
+
+Starting from version v38.0.0 onwards, all **open source installations** of CKEditor 5 (free users) carry a small “Powered by CKEditor” logo in the bottom right corner of the editing area. The label links directly to the [CKEditor website](https://ckeditor.com/). The new branding approach is designed to make sure the entire community knows who is powering and modernizing their rich text editor. You can [read more about it](https://github.com/ckeditor/ckeditor5/issues/14082) on GitHub.
```

Comment:
Maybe we should mention that logo will only be visible when editor is focused and only in the editable? (I know that users with our export features won't see it, but the ones with custom solutions will and may be concerned about it). Sorry if it was discussed already.

Reply:
I'll add it, thanks for bringing this up.